pkg/actions/identityproviders/wait.go

```diff
@@ -38,8 +39,5 @@ func (m *Manager) waitForUpdate(
 		clusterName,
 	)
 
-	if err := waiters.Wait(clusterName, msg, acceptors, newRequest, timeout, nil); err != nil {
```

Comment:
For these and the rest. These are empty ifs. I know that some prefer to return explicitly, but these are empty statements. If they would add logging or at least wrap the error, I would also leave them be. As such, they add fluff and extra code lines for no reason. :)